spring-xd-messagebus-kafka/src/main/java/org/springframework/xd/dirt/integration/kafka/KafkaMessageBus.java

```diff
@@ -789,4 +806,43 @@ private int roundRobin() {
 
 	}
 
+	@Override
+	public void storeForManualAck(MessageHeaders headers) {
+		String topic = (String) headers.get(KafkaHeaders.TOPIC);
+		Partition partition = new Partition(topic, (int) headers.get(KafkaHeaders.PARTITION_ID));
+		Long offset = (Long) headers.get(KafkaHeaders.OFFSET);
```

Comment:
yeah, but it is nice that you already have it addressed in the SI-kafka PR. Once that PR is merged, I can either update this PR or submit a new PR once the SI changes are merged.